cpp/src/robotick/workloads/control/SteeringMixerWorkload.cpp

```
@@ -38,7 +39,7 @@ namespace robotick
 		SteeringMixerOutputs outputs;
 		SteeringMixerConfig config;
 
-		void tick(const TickInfo&)
+		void tick(const TickInfo& tick)
```

Cppcheck (2.19.0)

[style] The function 'tick' is never used.

(unusedFunction)